Retry flaky UI tests in CI

mahendrapaipuri opened the pull request:
See [comment](https://github.com/mwouts/jupytext/issues/1260#issuecomment-2232440074)

Closes #1230

mwouts commented:
Hi @mahendrapaipuri , I am not sure this test is "flaky" - I had relaunched the publish CI multiple times and it failed on each attempt (https://github.com/mwouts/jupytext/actions/runs/9958694800/job/28138283717) - curious to learn what makes you think that retry would improve this?

mahendrapaipuri commented:
Well, if you look into the [CI runs of this PR](https://github.com/mwouts/jupytext/actions/runs/9969577431/job/27547063551#step:8:258), playwright actually marked that test as flaky. I guess the first run of that test is always a failure and it passes in the subsequent retries. 

Anyways, that test does not add any value and hence, I removed it. I added a workflow to update snapshots directly from a PR by commenting "update snapshots". Let me know what you think!

mwouts commented:
Thank you @mahendrapaipuri , this looks great! I will have to fix https://github.com/mwouts/jupytext/issues/1266 first but I am looking forward to taking your PR! Thanks
